src/ga4gh/vrs/_internal/models.py

```diff
@@ -262,6 +262,13 @@ class ReferenceLengthExpression(_ValueObject):
         None, description='The number of residues in the repeat subunit.'
     )
 
+    class ga4gh(_Ga4ghIdentifiableObject.ga4gh):
```

Comment:
Note: this class inheritance (GA4GH Identifiable) is a little confusing, since this object is _not_ GA4GH identifiable but _does_ use the GA4GH keys property to define its serialization. Not sure if it is useful to distinguish this in the VRS-Python class model, so just leaving as a general comment.

Reply:
Yeah the name may have been decided on before all the use cases were clear. I would be open to renaming `_Ga4ghIdentifiableObject` to `_Ga4ghDigestibleObject` or something else